measures/ResStockArguments/resources/constants.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Constants
-  def self.excludes
+  def self.build_residential_hpxml_excludes
```

Comment:
Should we be trying to obtain these values automatically rather than hard-coding this list?

Reply:
I'm not sure how to automatically identify these arguments. Do you have any ideas?

Reply:
I haven't fully thought this through, but can't you derive this list by basically comparing the arguments in the BuildResHPXML (or BuildResScheduleFile) measure to the ResStockArguments measure?

Reply:
You mean like in terms of their prefixes or something?

Reply:
Or maybe arguments for ResStockArguments could be all of BuildResHPXML's measure arguments, but optional?

Reply:
We decided to stick with what we have.